docker-compose.yml

```diff
@@ -38,6 +38,13 @@ services:
         labels:
             ca.mcgill.a11y.image.preprocessor: 2
             ca.mcgill.a11y.image.port: 5000
+    ner:
+        image: ner:latest
```

Comment:
Would not be this image tag, but I would need to create the CI/CD workflow file for this.

Reply:
Should I change it to `ner:unstable` ?